independent-publisher-connectors/OneNote (Personal)/readme.md

```diff
@@ -0,0 +1,29 @@
+# OneNote (Consumer)
+Microsoft OneNote is your digital notebook. One cross-functional notebook for all your notetaking needs.
+
+## Publisher: Troy Taylor, Hitachi Solutions
+
+## Prerequisites
+Use of this connector requires access to Microsoft Graph, which will require an Azure subscription with Azure Active Directory. Follow these [instructions](https://docs.microsoft.com/en-us/graph/auth-register-app-v2) to configure your app registration.
```

Comment:
@natalie-pienkowska FYI - This connector needs Graph API access, we probably need to create a first-party app to request access from the Graph API team and that would take some time like the other IP connectors we are working on.